exporter/metrics.go

```diff
@@ -296,6 +300,99 @@ func makeMetrics(prefix string, m bson.M, labels map[string]string, compatibleMo
 	return res
 }
 
+func extractHistograms(v []interface{}) map[string][]float64 {
+	if len(v) < 1 {
+		return nil
+	}
+
+	histograms := make(map[string][]float64)
+
+	for _, item := range v {
+		var s map[string]interface{}
+
+		switch i := item.(type) {
+		case map[string]interface{}:
+			s = i
+		case primitive.M:
+			s = map[string]interface{}(i)
+		default:
+			return nil
+		}
+
+		for key, value := range s {
+			switch i := value.(type) {
+			case int8:
+				histograms[key] = append(histograms[key], float64(i))
+			case int:
+				histograms[key] = append(histograms[key], float64(i))
+			case int32:
+				histograms[key] = append(histograms[key], float64(i))
+			case int64:
+				histograms[key] = append(histograms[key], float64(i))
+			case float32:
+				histograms[key] = append(histograms[key], float64(i))
+			case float64:
+				histograms[key] = append(histograms[key], float64(i))
+			}
+		}
+	}
+	// If all the items have the same number of fields, it is a histogram:
+	//         "histograms":  primitive.M{
+	//           "classicWorks": primitive.A{
+	//             primitive.M{
+	//               "count":      0,
+	//               "lowerBound": 0,
+	//             },
+	//             primitive.M{
+	//               "lowerBound": 128,
+	//               "count":      0,
+	//             },
+	//             primitive.M{
+	//               "lowerBound": 256,
+	//               "count":      0,
+	//             },
+	//           }
+	//         }
```

Comment:
I checked some histogram examples and I found out that each primitive.M contains two fields: some key and `count`. So I think that the key should be used as a label and `count` as a value.